### PR TITLE
Prevent glass break

### PR DESCRIPTION
**Pull request by iDirtniverse**
Test needed

### CODE REVIEW

**Comment by ItsLazer** [comment]
Test it on your server?

**Comment by iDirtniverse** [comment]
I'm on my phone sorry x(

**Comment by ItsLazer** [comment]
Is it the same comit as your server one because as soon as you join it doesn't set your gamemode straight away on your server.

**Comment by ItsLazer** [comment]
It's like Ike a 1-3 second time delay

**Comment by iDirtniverse** [comment]
You joined my server? My server doesn't have the update I just made, that's why I said test needed x3

**Comment by ItsLazer** [comment]
I'll go test it on my test servers

**Comment by iDirtniverse** [comment]
Updated, it works ;P

**Comment by ItsLazer** [comment]
I can't right now and @DriesBoy to do it

**Comment by ItsLazer** [comment]
Hold on is it updated to your server?

**Comment by iDirtniverse** [comment]
!! IGNORE FIRST COMMIT, THE REST ARE TYPOS SORRY!!

Typos from phone :(

Tested, works

File: SW/src/svile/sw/SWarena.php [hunk 1]
```diff
@@ -521,12 +521,15 @@ private function start()
                 if ($p->getAttributeMap() != null) {//just to be really sure
                     $p->setHealth($this->pg->configs['join.health']);
                     $p->setFood(20);
+                    $p->setGamemode(Player::SURVIVAL);
```
Comment (svilex) [inline SW/src/svile/sw/SWarena.php:524]: why is this inside the if?

File: SW/src/svile/sw/SWarena.php [hunk 2]
```diff
@@ -521,12 +521,15 @@ private function start()
...
                 }
                 $p->sendMessage($this->pg->lang['game.start']);
                 if ($p->getLevel()->getBlock($p->floor()->subtract(0, 2))->getId() == 20)
                     $p->getLevel()->setBlock($p->floor()->subtract(0, 2), Block::get(0), true, false);
+
                 if ($p->getLevel()->getBlock($p->floor()->subtract(0, 1))->getId() == 20)
                     $p->getLevel()->setBlock($p->floor()->subtract(0, 1), Block::get(0), true, false);
+
```
Comment (svilex) [inline SW/src/svile/sw/SWarena.php:532]: thanks, but, i don't need new lines here

File: SW/src/svile/sw/SWarena.php (hunk 1 above)
Comment (iDirtniverse) [reply SW/src/svile/sw/SWarena.php:524]: ? I think thats where the game joined function was... ?

File: SW/src/svile/sw/SWarena.php (hunk 2 above)
Comment (iDirtniverse) [reply SW/src/svile/sw/SWarena.php:532]: and I dont think I added those, I only add when game has started that set player back to be able to break blocks aka survival mode

Comment (dries-c) [reply SW/src/svile/sw/SWarena.php:532]: you used the webeditor thats the  problem

File: SW/src/svile/sw/SWarena.php (hunk 1 above)
Comment (svilex) [reply SW/src/svile/sw/SWarena.php:524]: No, this function is called when the game starts.
Setting the gamemode to survival here (in this function) is right. but what i asked you is why setGanemode() is inside this `if ($p->getAttributeMap() != null)` if

Comment (iDirtniverse) [reply SW/src/svile/sw/SWarena.php:524]: Oo, okay, sorry senpai

File: SW/src/svile/sw/SWarena.php (hunk 2 above)
Comment (iDirtniverse) [reply SW/src/svile/sw/SWarena.php:532]: @Driesboy yes I was

**Comment by lakub799** [comment]
Plz Fix 1.0.7 Plzzzzzz T^T

**Comment by iDirtniverse** [comment]
Does the prevention not work for 1.0.7? Did u add the additional code to it?

**Comment by ItsLazer** [comment]
It actually doesnt work anymore for some reason

**Pull request closed without merging**